### ISSUE TITLE
Dropdown: incorrect type definition for `additionLabel`

### ISSUE THREAD

**Issue by DavidHall** [Semantic-Org/Semantic-UI-React#3054]
### Steps
* When using typescript use the following code from the `Dropdown` component samples on using a component for the dropdown additionLabel:

```
      <Dropdown
        options={this.state.options}
        placeholder='Choose Language'
        search
        selection
        fluid
        allowAdditions
        additionLabel={<i style={{ color: 'red' }}>Custom Language: </i>}
        value={currentValue}
        onAddItem={this.handleAddition}
        onChange={this.handleChange}
      />
```
* Compile using the typescript compiler

### Expected Result
Example compiles correctly, giving a dropdown component with a component as the additionLabel.

### Actual Result
Typescript compilation fails with the following error:

```
error TS2322: Type '{ additionLabel: Element; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<Component<DropdownProps, ComponentState>> & Readon...'.
  Type '{ additionLabel: Element; }' is not assignable to type 'Readonly<DropdownProps>'.
    Types of property 'additionLabel' are incompatible.
      Type 'Element' is not assignable to type 'string | number'.
        Type 'Element' is not assignable to type 'number'.
```

### Version
0.82.1

### Testcase
I'm not sure if it possible to create a test case in a pen that demonstrates a Typescript compiler error.

**Comment by DavidHall** [comment]
I have a pull request ready with a fix for this, but hesitated on issuing it for two reasons:
* The current `Dropdown.d.ts` definition for the `additionLabel`  prop is:

      additionLabel?: number | string;

but the prop types are:

    additionLabel: PropTypes.oneOfType([
      PropTypes.element,
      PropTypes.string,
    ]),

Therefore I was unsure if:

    additionLabel?: number | string | React.ReactNode;

or

    additionLabel?: string | React.ReactNode;

was the better fix.

* I was not certain how to write tests Typescript compilation in the project. I added a reference to `Dropdown` in `typings.tsx`, that would demonstrate the error when `tsc` is run, but there doesn't seem to be project command that runs tsc.

**Comment by brianespinosa** [comment]
```
additionLabel?: number | string | React.ReactNode;
```
@DavidHall this is probably correct. I think you can go ahead and submit a PR for this and someone who has more TS experience than I do can confirm on the PR. It will probably get merged faster if conversation can happen over code in a PR. 👍

**Comment by layershifter** [comment]
Was fixed in #3055 :+1:
